apps/frontpage/components/home/manager/component-controls.tsx

```diff
@@ -0,0 +1,276 @@
273.95 425V417.224H275.246V425H273.95ZM280.629 425.144C279.936 425.144 279.328 424.979 278.805 424.648C278.293 424.317 277.893 423.853 277.605 423.256C277.328 422.648 277.189 421.933 277.189 421.112C277.189 420.28 277.328 419.565 277.605 418.968C277.893 418.36 278.293 417.896 278.805 417.576C279.328 417.245 279.936 417.08 280.629 417.08C281.237 417.08 281.77 417.213 282.229 417.48C282.698 417.747 283.045 418.115 283.269 418.584V413.72H284.565V425H283.301V423.576C283.077 424.067 282.73 424.451 282.261 424.728C281.792 425.005 281.248 425.144 280.629 425.144ZM280.901 424.104C281.626 424.104 282.208 423.848 282.645 423.336C283.082 422.813 283.301 422.072 283.301 421.112C283.301 420.141 283.082 419.4 282.645 418.888C282.208 418.376 281.626 418.12 280.901 418.12C280.165 418.12 279.578 418.376 279.141 418.888C278.714 419.4 278.501 420.141 278.501 421.112C278.501 422.072 278.714 422.813 279.141 423.336C279.578 423.848 280.165 424.104 280.901 424.104ZM290.501 425.144C289.275 425.144 288.304 424.787 287.589 424.072C286.875 423.347 286.517 422.365 286.517 421.128C286.517 420.328 286.677 419.624 286.997 419.016C287.317 418.397 287.755 417.923 288.309 417.592C288.875 417.251 289.525 417.08 290.261 417.08C291.317 417.08 292.144 417.421 292.741 418.104C293.339 418.776 293.637 419.704 293.637 420.888V421.384H287.781C287.824 422.269 288.08 422.947 288.549 423.416C289.019 423.875 289.669 424.104 290.501 424.104C290.971 424.104 291.419 424.035 291.845 423.896C292.272 423.747 292.677 423.507 293.061 423.176L293.509 424.088C293.157 424.419 292.709 424.68 292.165 424.872C291.621 425.053 291.067 425.144 290.501 425.144ZM290.293 418.04C289.557 418.04 288.976 418.269 288.549 418.728C288.123 419.187 287.872 419.789 287.797 420.536H292.517C292.485 419.747 292.277 419.133 291.893 418.696C291.52 418.259 290.987 418.04 290.293 418.04ZM298.137 425.144C297.529 425.144 296.953 425.059 296.409 424.888C295.865 424.717 295.417 424.477 295.065 424.168L295.481 423.256C295.865 423.565 296.286 423.795 296.745 423.944C297.214 424.083 297.689 424.152 298.169 424.152C298.777 424.152 299.236 424.04 299.545 423.816C299.854 423.592 300.009 423.288 300.009 422.904C300.009 422.605 299.908 422.371 299.705 422.2C299.502 422.019 299.182 421.88 298.745 421.784L297.289 421.464C295.966 421.176 295.305 420.493 295.305 419.416C295.305 418.712 295.582 418.147 296.137 417.72C296.702 417.293 297.438 417.08 298.345 417.08C298.878 417.08 299.385 417.165 299.865 417.336C300.356 417.496 300.761 417.736 301.081 418.056L300.649 418.968C300.34 418.68 299.977 418.461 299.561 418.312C299.156 418.152 298.75 418.072 298.345 418.072C297.748 418.072 297.294 418.189 296.985 418.424C296.676 418.659 296.521 418.968 296.521 419.352C296.521 419.949 296.916 420.333 297.705 420.504L299.161 420.808C299.844 420.957 300.361 421.197 300.713 421.528C301.065 421.848 301.241 422.285 301.241 422.84C301.241 423.555 300.958 424.12 300.393 424.536C299.828 424.941 299.076 425.144 298.137 425.144ZM302.93 425V423.368H304.562V425H302.93Z"
```

Comment:
We can ship this as-is. But the next time we update this, we should look into using real text, to avoid these giant `path`s.

Reply:
We can't use real text since this is not a font we support. It felt cleaner to do it that way instead of importing a font for the full site.